test/e2e_tests/sync.go

```diff
@@ -54,87 +84,138 @@ var _ = Describe("Validate the placement sync controller", func() {
 		By("Checks if ManifestWork was deleted")
 		manifestWork := &workv1.ManifestWork{}
 		Eventually(func() bool {
-			mwName := NamespaceManifestWorkPrefix + assertionCapp.Namespace + "-" + assertionCapp.Name
+			mwName := namespaceManifestWorkPrefix + assertionCapp.Namespace + "-" + assertionCapp.Name
 			Expect(k8sClient.Get(context.Background(), client.ObjectKey{Name: mwName, Namespace: mwNamespace}, manifestWork)).Should(Succeed())
 			return utilst.DoesResourceExist(k8sClient, manifestWork)
 		}, testconsts.Timeout, testconsts.Interval).ShouldNot(BeFalse())
 	})
 
-	It("Should copy the secret from volumes to ManifestWork ", func() {
+	It("Should copy the secret from volumes to ManifestWork", func() {
 		baseSecret := mock.CreateSecret()
 		secret := utilst.CreateSecret(k8sClient, baseSecret)
 		baseCapp := mock.CreateBaseCapp()
-		baseCapp.Spec.ConfigurationSpec.Template.Spec.Volumes = append(baseCapp.Spec.ConfigurationSpec.Template.Spec.Volumes, corev1.Volume{
-			Name: secret.Name,
-			VolumeSource: corev1.VolumeSource{
-				Secret: &corev1.SecretVolumeSource{
-					SecretName: secret.Name,
+		baseCapp.Spec.ConfigurationSpec.Template.Spec.Volumes = append(
+			baseCapp.Spec.ConfigurationSpec.Template.Spec.Volumes,
+			corev1.Volume{
+				Name: secret.Name,
+				VolumeSource: corev1.VolumeSource{
+					Secret: &corev1.SecretVolumeSource{
+						SecretName: secret.Name,
+					},
 				},
-			},
-		})
+			})
 
-		baseCapp.Spec.ConfigurationSpec.Template.Spec.Containers[0].VolumeMounts = append(baseCapp.Spec.ConfigurationSpec.Template.Spec.Containers[0].VolumeMounts, corev1.VolumeMount{
-			Name:      secret.Name,
-			MountPath: "/test/mount",
-		})
-
-		desiredCapp := utilst.CreateCapp(k8sClient, baseCapp)
+		baseCapp.Spec.ConfigurationSpec.Template.Spec.Containers[0].VolumeMounts = append(
+			baseCapp.Spec.ConfigurationSpec.Template.Spec.Containers[0].VolumeMounts,
+			corev1.VolumeMount{
+				Name:      secret.Name,
+				MountPath: "/test/mount",
+			})
 
-		By("Checks unique creation of Capp")
-		assertionCapp := utilst.GetCapp(k8sClient, desiredCapp.Name, desiredCapp.Namespace)
-		Expect(assertionCapp.Name).ShouldNot(Equal(baseCapp.Name))
+		verifySecretOrConfigMapCopy(baseCapp, secret.Name, secret.Namespace, kindSecret)
```

Comment:
why to hold secret and configMap as variable v1 already have the variables in the kind